software/conduwuit.yml

```diff
@@ -0,0 +1,10 @@
+name: conduwuit
+website_url: https://conduwuit.puppyirl.gay
+description: conduwuit is a well-maintained, featureful, hard-fork of Conduit with tons of new features, many bug fixes, huge performance improvements, quality of life enhancements, moderation tools, and much more!.
```

Comment:
The description should explain what the software does

Reply:
It mentions Conduit, yeah but I guess it really should explain what it is. Like a matrix server or something. I see then.